CMakeLists.txt

```diff
@@ -114,6 +120,7 @@ set(STLINK_SOURCE
         src/stlink-lib/md5.c
         src/stlink-lib/sg.c
         src/stlink-lib/usb.c
+		src/stlink-lib/helper.c
```

Comment:
wrong indentation.

Reply:
I fixed it by addition commit 1bbdd74. It appears in the Commits tab, but its changes are not applied to the Files changed tab. I do not know why it is so.

Reply:
you dit it for the helper.**h**, please do it for the helper.**c** also.

Reply:
oops, sorry. I fixed it too.